plugins/dialog/dialogDefinition.js

```diff
@@ -1053,8 +1053,8 @@
  *
  * If the direction was set, a marker will be prepended to every non-empty value of this input:
  *
- * * [`\u202A`](http://unicode.org/cldr/utility/character.jsp?a=202A) &ndash; for Right-To-Left,
- * * [`\u202B`](http://unicode.org/cldr/utility/character.jsp?a=202B) &ndash; for Left-To-Right.
+ * * [`\u202A`](https://www.unicode.org/charts/PDF/U2000.pdf) &ndash; for Right-To-Left,
+ * * [`\u202B`](https://www.unicode.org/charts/PDF/U2000.pdf) &ndash; for Left-To-Right.
```

Comment:
PDF references aren't cool as they are not very specific (you still need to find the interesting part). But in this case I'm afraid we don't have a better option - I did find references on some different websites (like https://charbase.com/202a-unicode-left-to-right-embedding or https://www.fileformat.info/info/unicode/char/202a/index.htm), but none of them has any connection to Unicode so they may shut down any moment and we will run into the same issue again. Still, with the proposed solution we are vulnerable to the simple change of filename :see\_no\_evil: But let's hope Unicode forgets about this file and never changes it :smile: